pulsar-functions/utils/src/main/java/org/apache/pulsar/functions/utils/Reflections.java

```diff
@@ -153,8 +153,9 @@ public static Object createInstance(String userClassName,
     public static Object createInstance(String userClassName, java.io.File jar) {
         try {
             return createInstance(userClassName, loadJar(jar));
-        } catch (Exception ex) {
-            return null;
+        } catch (MalformedURLException e) {
```

Comment:
How about removing the try catch block and declaring createInstance to throw exception. I believe that the upper layer will print this exception. That way we will not have duplicate messages